packages/test/test-service-load/src/optionsMatrix.ts

```
@@ -25,10 +25,29 @@ const loaderOptionsMatrix: OptionsMatrix<ILoaderOptions> = {
     summarizeProtocolTree: [undefined],
 };
 
-export const generateLoaderOptions = (seed: number): ILoaderOptions[]=>
-    generatePairwiseOptions<ILoaderOptions>(
-        loaderOptionsMatrix,
+export function applyOverrides<T>(options: OptionsMatrix<T>, optionsOverrides: Partial<OptionsMatrix<T>> | undefined) {
+    const realOptions: OptionsMatrix<T> = {...options};
+    if(optionsOverrides !== undefined) {
+        for(const key of Object.keys(optionsOverrides)) {
+            const override = optionsOverrides[key];
+            if(override !== undefined) {
+                if(Array.isArray(override)) {
+                    realOptions[key] =  override;
+                }else{
+                    console.error(`Override for ${key} is not array: ${JSON.stringify(optionsOverrides)}`);
```

I'd prefer fail fast (exception) over console logs